app/src/main/kotlin/no/nav/tiltakspenger/vedtak/routes/behandling/dto/BehandlingIdDTO.kt

```diff
@@ -0,0 +1,5 @@
+package no.nav.tiltakspenger.vedtak.routes.behandling.dto
+
+data class BehandlingIdDTO(
+    val id: String,
```

Comment:
Ikke noe du trenger gjøre nå, men kanskje vi skal fjerne denne og heller alltid returnere minst behandlingen? Også kan frontend plukke ut kun id hvis den ikke trenger mer

Reply:
Har tenkt litt det samme, bør generelt returnere det som trengs for state-oppdateringer i frontend. Må skrive om en del i frontend for å dra nytte av det, men kan ta det litt og litt.